src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -127,7 +127,7 @@ public bool ReportingAnalytics
         /// <summary>
         /// Return if Google Analytics Client is allowed to send analytics info
         /// </summary>
-        private bool ReportingGoogleAnalytics
+        public bool ReportingGoogleAnalytics
```

Comment:
why public?

Reply:
I needed to make this public to satisfy the changes to `IAnalyticsClient.cs` and to avoid adding new references.